### ISSUE TITLE
Update CLI Texts

### ISSUE THREAD

**Issue by lukaswolfrum** [amosproj/amos2022ss07-nft-playbook#51]
## User story
1. As a power user
2. I want / need intuitively understand in which way i have to run through the program
3. So that I an create an NFT fast and without barriers

## Acceptance criteria
* Edit the existing menu points and texts according to the attached file

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by lukaswolfrum** [comment]
[Screen Mock Up 2.0](https://docs.google.com/document/d/1AA9mTvmyeRGY0EZUkIbIvz1FcFJUV9nKlPYVlxITKSI/edit)

Hey guys, I found our other Screen Mock Up to be quite confusing therefor I created a new one. I hope this is much clearer. I case you have any questions let me know.
